### PR TITLE
feat(CritiqueBrainz): add presence

### PR DESCRIPTION
**Pull request by theusaf**
## Description 
Adds a presence for CritiqueBrainz.
This PR is a part of #5460

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Review by EncryptedDev — APPROVED** [review]

File: websites/C/CritiqueBrainz/presence.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+					presenceData.details = "Writing a review";
+					presenceData.state =
+						document.querySelector<HTMLAnchorElement>("dd a").textContent;
+					break;
+				}
+				default: {
+					if (/^\w{8}-\w{4}-\w{4}-\w{4}-\w{12}$/.test(pathSplit[1])) {
+						const image =
+							document.querySelector<HTMLImageElement>(".cover-art");
+						if (image) presenceData.largeImageKey = image.src;
```
Comment (DarkVillager) [inline websites/C/CritiqueBrainz/presence.ts:87]:
```suggestion
presenceData.largeImageKey = document.querySelector<HTMLImageElement>(".cover-art")?.src ?? Assets.Logo;
```

File: websites/C/CritiqueBrainz/presence.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
+const presence = new Presence({
+		clientId: "1017821399705985097",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
```
Comment (DarkVillager) [inline websites/C/CritiqueBrainz/presence.ts:5]:
```suggestion
enum Assets {
	Logo = "https://i.imgur.com/LG31f5s.png",
}

```

File: websites/C/CritiqueBrainz/presence.ts [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/LG31f5s.png",
```
Comment (DarkVillager) [inline websites/C/CritiqueBrainz/presence.ts:8]:
```suggestion
			largeImageKey: "Assets.Logo",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/C/CritiqueBrainz/presence.ts (hunk 2 above)
Comment (EncryptedDev) [reply websites/C/CritiqueBrainz/presence.ts:5]: Not needed for one asset

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/C/CritiqueBrainz/presence.ts (hunk 1 above)
Comment (DarkVillager) [reply websites/C/CritiqueBrainz/presence.ts]: @theusaf unresolved.

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**